deployment/stacks/service/overlays/dev/template/kustomization.yaml

```diff
@@ -1,8 +1,8 @@
 bases:
   - ../../base
 
-patchesStrategicMerge:
-  - uec-dos-api-cs-proxy-ingress-customisation.yaml
+#patchesStrategicMerge:
```

Comment:
Should we remove this altogether now we know it fixes the problem

Reply:
Yes, will action